apps/nextjs-website/src/components/organisms/FutureWebinarsShowcase/FutureWebinarsShowcase.tsx

```diff
@@ -0,0 +1,108 @@
+'use client';
+import { Webinar } from '@/lib/types/webinar';
+import { Alert, Box, Snackbar, Typography, useTheme } from '@mui/material';
+import React, { useState } from 'react';
+import LinkButton from '@/components/atoms/LinkButton/LinkButton';
+import EContainer from '@/editorialComponents/EContainer/EContainer';
+import { useUser } from '@/helpers/user.helper';
+import { snackbarAutoHideDurationMs } from '@/config';
+import WebinarCard from '@/components/molecules/WebinarCard/WebinarCard';
+import { useTranslations } from 'next-intl';
+
+export type FutureWebinarsShowcaseProps = {
+  link?: { href?: string; label: string };
+  title?:
+    | 'dontLoseNext'
+    | 'dontLoseNextPlural'
+    | 'next'
+    | 'our'
+    | 'participateTo';
+  webinars: Webinar[];
+};
+
+const FutureWebinarsShowcase = ({
+  link,
+  title = 'our',
+  webinars,
+}: FutureWebinarsShowcaseProps) => {
+  const theme = useTheme();
+  const t = useTranslations('webinar.webinarsSection');
+  const [error, setError] = useState<string | null>(null);
+  const { aligned: userAligned } = useUser();
+
+  return (
+    <Box
+      py={5}
+      sx={{
+        backgroundImage: 'url(/images/webinars.png)',
+        backgroundRepeat: 'no-repeat',
+        backgroundSize: 'cover',
+        backgroundPosition: 'bottom left',
+        paddingY: 8,
+        position: 'relative',
+        '&::before': {
+          content: '""',
+          position: 'absolute',
+          top: 0,
+          right: 0,
+          bottom: 0,
+          left: 0,
+          backgroundColor: 'rgba(0, 115, 230, 0.7)',
+        },
+      }}
+    >
+      <EContainer>
+        <Box sx={{ display: 'flex', flexDirection: 'column' }}>
```

Comment:
I see an issue for webinar with short title and descriptions, this fix the problem:
```suggestion
        <Box sx={{ display: 'flex', flexDirection: 'column', width: '100%' }}>
```